egs/bentham/v1/local/make_features.py

```diff
@@ -0,0 +1,266 @@
+#!/usr/bin/env python3
```

Comment:
Thanks, I guess as compared to the egs/iam/make_features, this script is converting a colored image into grayscale but currently is not using the slant and skew correction augmentations specific to iam.  If there is a slant/skew in the text line images then the slant and skew correction can be very helpful.

If the slant/skew correction changes did not help, then similar to ASR setup, for hwr setup,  idea is to use a common script for feature extraction.  Can you please modify the egs/cifar/v1/image/ocr/make_features.py script to convert the colored image into grayscale if the number of channels passed is 1 and call it here.

Reply:
Do you mean I should have a soft link to egs/cifar/v1/image/ocr/make_features.py? But Dan said the scripts in local are usually independent so changes in recipes can be made without affecting other recipes.